service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -97,52 +132,65 @@ public PolicyEvaluationResult evalPolicy(final PolicyEvaluationRequestV1 request
             } else {
                 supplementalSubjectAttributes = new HashSet<>(request.getSubjectAttributes());
             }
-            result = evalPolicy(request.getResourceIdentifier(), request.getSubjectIdentifier(), request.getAction(),
-                    supplementalResourceAttributes, supplementalSubjectAttributes);
+
+            for (PolicySet policySet : filteredPolicySets) {
+                result = evalPolicySet(policySet, subjectIdentifier, uri, action, supplementalResourceAttributes,
+                        supplementalSubjectAttributes);
+                if (result.getEffect() != Effect.NOT_APPLICABLE) {
```

Comment:
Stylistic quibble. I find the following more readable and explicit:
```if (result.getEffect() == Effect.NOT_APPLICABLE) continue;```
This way you don't even need the comment below.